src/app/components/icons/IconDrawer.tsx

```diff
@@ -150,16 +172,82 @@ export const IconDrawer: React.FC = () => {
                             >
                                 Download
                             </Typography>
+                            <div>
+                                <FormControl className={classes.formControl}>
+                                    <InputLabel id="icon-size-select-label">
+                                        Select a Size:
+                                    </InputLabel>
+                                    {selectedIcon.isMaterial ? (
+                                        <Select
+                                            labelId="icon-size-select-label"
+                                            id="icon-size-select"
+                                            value={iconSize}
+                                            onChange={handleChange}
+                                        >
+                                            <MenuItem value={18}>18dp</MenuItem>
+                                            <MenuItem value={24}>24dp</MenuItem>
+                                            <MenuItem value={36}>36dp</MenuItem>
+                                            <MenuItem value={48}>48dp</MenuItem>
+                                        </Select>
+                                    ) : (
+                                        <Select
+                                            labelId="icon-size-select-label"
+                                            id="icon-size-select"
+                                            value={iconSize}
+                                            onChange={handleChange}
+                                        >
+                                            <MenuItem value={24}>24dp</MenuItem>
+                                            <MenuItem value={48}>48dp</MenuItem>
+                                        </Select>
```

Comment:
instead of conditionally rendering the entire Select, you can conditionally render the two extra menu items:

```
<Select
    labelId="icon-size-select-label"
    id="icon-size-select"
    value={iconSize}
    onChange={handleChange}
>
    {selectedIcon.isMaterial && <MenuItem value={18}>18dp</MenuItem>}
    <MenuItem value={24}>24dp</MenuItem>
    {selectedIcon.isMaterial && <MenuItem value={36}>36dp</MenuItem>}
    <MenuItem value={48}>48dp</MenuItem>
</Select>
```

Reply:
Same for the color picker